src/scss/partials/_skipToMain.scss

```diff
@@ -0,0 +1,8 @@
+.skipTo {
```

Comment:
If you want this to respect a `margin-top` property of `4px`, think you would also need to make the `skipTo` link `display: block` else it'll only get the side margins.